presto-clickhouse/src/test/java/io/prestosql/plugin/clickhouse/TestClickHouseDistributedQueries.java

```diff
@@ -0,0 +1,203 @@
+                1);
+        assertUpdate("INSERT INTO " + tableName + " (date_column) VALUES (TIMESTAMP '2019-11-18 22:13:40')", 1);
+
+        assertQuery("SELECT * FROM " + tableName,
+                "VALUES " + "(1, 2, 3, 4, NULL, NULL, NULL, NULL), "
+                        + "(NULL, NULL, NULL, NULL, 'aa     ', 'aa     ', 'aa     ', NULL), "
+                        + "(NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL), "
+                        + "(NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL), "
+                        + "(NULL, NULL, NULL, NULL, NULL, NULL, NULL, DATE '2019-11-18')");
+
+        assertQueryFails("INSERT INTO " + tableName + " (integer_column) VALUES (3e9)",
+                "Out of range for integer: 3.0E9");
+
+        assertUpdate("DROP TABLE " + tableName);
+    }
+
+    @Override
+    protected boolean supportsArrays()
+    {
+        return false;
+    }
+
+    @Override
+    protected boolean supportsCommentOnTable()
+    {
+        return false;
+    }
+
+    @Override
+    public void testCommentColumn()
+    {
+        // currently does not support
+    }
+
+    @Override
+    public void testRenameTable()
+    {
+        String tableName = "test_rename_" + randomTableSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " (x int)");
+        assertUpdate("INSERT INTO " + tableName + " VALUES (123)", 1);
+
+        String renamedTable = "test_rename_new_" + randomTableSuffix();
+        assertUpdate("ALTER TABLE " + tableName + " RENAME TO " + renamedTable);
+        assertQuery("SELECT x FROM " + renamedTable, "VALUES 123");
+
+        String testExistsTableName = "test_rename_new_exists_" + randomTableSuffix();
+        assertUpdate("ALTER TABLE IF EXISTS " + renamedTable + " RENAME TO " + testExistsTableName);
+        assertQuery("SELECT x FROM " + testExistsTableName, "VALUES 123");
+
+        String uppercaseName = "TEST_RENAME_" + randomTableSuffix(); // Test an upper-case, not delimited identifier
+        assertUpdate("ALTER TABLE " + testExistsTableName + " RENAME TO " + uppercaseName);
+        assertQuery("SELECT x FROM " + uppercaseName.toLowerCase(ENGLISH), // Ensure select allows for lower-case, not
+                // delimited identifier
+                "VALUES 123");
+
+        assertUpdate("DROP TABLE " + uppercaseName);
+
+        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
+        assertFalse(getQueryRunner().tableExists(getSession(), renamedTable));
+
+        assertUpdate("ALTER TABLE IF EXISTS " + tableName + " RENAME TO " + renamedTable);
+        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
+        assertFalse(getQueryRunner().tableExists(getSession(), renamedTable));
+    }
+
+    @Override
+    public void testRenameColumn()
+    {
+        // currently does not support
+    }
+
+    @Override
+    public void testDropColumn()
+    {
+        // currently does not support
```

Comment:
This overrides should be removed or enhanced with an explanation why.

The recommended way of disabling a test is using
```
assertThatThrownBy(super::
 ...
throw new SkipException(
```

like here
https://github.com/prestosql/presto/blob/cbbc6aaf699e36223aa2d9832928e9e54b80054b/presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/BaseCassandraDistributedQueries.java#L94-L101